src/layouts/RootContainer.jsx

```diff
@@ -3,15 +3,15 @@ import styled from "styled-components";
 
 const StyledRoot = styled.div`
   width: 100%;
-  height: 100%;
+  min-height: 100%;
```

Comment:
из-за этого у тебя на пустой странице прокрутка лишняя 
```suggestion
  min-height: calc(100% - [высота твоего хэдера];
```

Reply:
я бы может даже предложила скролл перенести в компонент со списком айтемов